fields/src/to_field_vec.rs

```diff
@@ -77,7 +83,7 @@ impl<F: PrimeField> ToConstraintField<F> for [u8] {
     #[inline]
     fn to_field_elements(&self) -> Result<Vec<F>, ConstraintFieldError> {
         // Derive the field size in bytes, floored to be conservative.
-        let floored_field_size_in_bytes = (<F as PrimeField>::Parameters::CAPACITY / 8) as usize;
+        let floored_field_size_in_bytes = (F::size_in_data_bits() / 8) as usize;
```

Comment:
Where is this defined?

Reply:
`F::size_in_data_bits()` is a trait method that returns `<F as PrimeField>::Parameters::CAPACITY`.

Reply:
The way we pack the `[u8]` isn't my favorite, however I tried modifying it, and couldn't get it to work in the brief attempt I made. The current changes here are merely for readability, the logic remains unchanged